osmo-test-5/osmo-test-5.assetlist.json

```diff
@@ -215,6 +215,43 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/marstestnet/images/mars.svg"
       }
+    },
+    {
+      "description": "The native utility token of the KYVE network.",
+      "denom_units": [
+        {
+          "denom": "ibc/AB8AF05799E299FB5C5C80781DA35887F53E029745D20E5641233DB4E6B28515",
+          "exponent": 0,
+          "aliases": [
+            "tkyve"
+          ]
+        },
+        {
+          "denom": "kyve",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/AB8AF05799E299FB5C5C80781DA35887F53E029745D20E5641233DB4E6B28515",
+      "name": "KYVE Testnet",
+      "display": "kyve",
+      "symbol": "KYVE",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "kaon",
```

Comment:
I think this is referencing kyvetestnet (or could be kyvedevnet?). Please correct, as 'kaon' is not a registered chain name in the Chain Registry.